dfinity/sdk
# feat: add open flag to dfx deploy command

# Description

This pull request adds a new feature to the dfx deploy command where users can specify the --open flag to automatically open the URL of the deployed canisters in the default browser of the machine. This enhancement aims to improve user experience by providing a convenient way to access the deployed canisters immediately after deployment.

# How Has This Been Tested?

To test this feature:

- Ensure the dfx command-line tool is installed and configured properly.
- Run the dfx deploy --open command with a project that has deployable canisters.
- Verify that after deployment, the default browser opens with the URL of the deployed canisters.

# Checklist:

- [x] The title of this PR complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/).
- [x] I have edited the CHANGELOG accordingly.
